docker/Dockerfile

```diff
@@ -1,11 +1,18 @@
-FROM iron/go:dev
+FROM golang:buster
 
-ENV SRC_DIR=/app
+WORKDIR /app
+COPY . /app
 
-WORKDIR $SRC_DIR
+RUN go get
+RUN go build
+RUN chmod +x Valhalla
 
-ADD . $SRC_DIR
+RUN echo "#!/bin/sh" > docker-entrypoint.sh && \
+  echo "set -ex" >> docker-entrypoint.sh && \
+  echo 'exec "$@"' >> docker-entrypoint.sh && \
+  chmod +x docker-entrypoint.sh
 
-RUN cd $SRC_DIR; go build -o server
+EXPOSE 8484 8585 8584 8685
```

Comment:
No need to expose ports here as the compose overrides.

Reply:
Its best practice to expose ports in Dockerfile so that one can understand what ports an app exposes without going through any app code.
I will expose relevant ports however.

Reply:
I agree with the statement, but in this case the app exposes ports based on config which isn't baked into the image. Therefore, I feel it is best for a user to manually expose the ports they have determined via compose or standard docker command line.

Reply:
Makes sense, will update accordingly :)